lib/suspenders/generators/stylesheet_base_generator.rb

```diff
@@ -23,9 +23,8 @@ def install_bitters
       run "bitters install --path app/assets/stylesheets"
     end
 
-    # TODO: This is breaking the test suite
     def install_normalize_css
-      run "bin/yarn add normalize.css"
+      run "yarn add normalize.css"
```

Comment:
Note: This was working because we were using [Webpacker](https://github.com/rails/webpacker/blob/master/lib/install/bin/yarn).